webapp/users.py

```diff
@@ -3,9 +3,25 @@
 # to temporarily disable non-crucial pylint errors in conformity
 # pylint: disable=invalid-name
 
+import pymysql
+import os
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
 from flask_login import UserMixin, AnonymousUserMixin, LoginManager
 
+app = Flask(__name__)
+
+# put them all together as a string that shows SQLAlchemy where the database is
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://UserAdmin:^0o9JpRAPazf@webapp-dev-db.c1cjueiaoepn.us-west-1.rds.amazonaws.com:3306/user.accounts'
```

Comment:
Let's put this string as a server config option, or at least not hardcoded (??). I don't think that it's secure to leave DB strings out in the wild.